setup.py

```diff
@@ -32,6 +32,7 @@ def has_environment_marker_platform_impl_support():
     'zope.interface>=4.1.3',
     'protego>=0.1.15',
     'itemadapter>=0.1.0',
+    'fuzzywuzzy==0.18.0',
```

Comment:
_If_ we add this dependency, we should not hardcode a specific version. Unlike lock files, `setup.py` is meant to indicate  a range of versions known to work.

Reply:
I deleted that dependency in favor to use a builtin one(`difflib`)